src/dmd/expressionsem.d

```diff
@@ -5130,6 +5130,13 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
 
     override void visit(DeleteExp exp)
     {
+        // @@@DEPRECATED_2019-02@@@
+        // 1. Deprecation for 1 year
+        // 2. Error for 1 year
+        // 3. Removal, "delete" can be used for other identities
+        if (!exp.isRAII)
```

Comment:
It's important that when the deprecation period is over, the keyword is removed from the parser.  Please add a comment about that here so it doesn't get overlooked.

Reply:
added "keyword" to the comment.